ui/component/walletExport/view.jsx

```diff
@@ -0,0 +1,75 @@
+// @flow
+import React from 'react';
+import Button from 'component/button';
+import { Form, FormField } from 'component/common/form';
+import Card from 'component/common/card';
+import Lbry from 'lbry';
+import { clipboard } from 'electron';
+import * as ICONS from 'constants/icons';
+import FileExporter from 'component/common/file-exporter';
+
+type Props = {
+  toast: (string, boolean) => void,
+};
+
+function WalletExport(props: Props) {
+  const { toast } = props;
+
+  const [password, setPassword] = React.useState();
+  const [data, setData] = React.useState();
+  const [fetching, setFetching] = React.useState();
+  const getData = async (password) => {
+    setFetching(true);
+    const data = await Lbry.sync_apply({ password });
```

Comment:
- [ ] `password` doesn't seem to impact the exported data?

I entered 2 different passwords, but the copied data is identical for both.

Reply:
Oh, I misunderstood the purpose of `password` -- it's not for encrypting the output, but to decrypt the wallet (if a password exists), then merge (with nothing), then return the data.

Reply:
Wait, really?

> * [ ]  `password` doesn't seem to impact the exported data?
> 
> I entered 2 different passwords, but the copied data is identical for both.

This shouldn't be the case - I did sync apply <password> by cli and got two different hashes and different data. Will check.

Reply:
> 

I really think it's for encrypting the output / decrypting it on import. You always need some string for the password.

Reply:
man ... read the docs and and tried again.
False alarm ... all good.